integration/src/main/resources/configs/integration/ControlledAzureVmNoPublicIpLifecycle.json

```diff
@@ -0,0 +1,23 @@
+{

```

Comment:
As a heads-up, these tests will be runnable locally but won't be included as part of the automated nightly/PR integration test runs unless you add them to the FullIntegration test suite (`integration/src/main/resources/suites/FullIntegration.json`), not sure if you want these tests included there or not. I also don't know if Azure is enabled in the WSM nightly test environment (`wsmtest`), which would be another issue.

Reply:
That's good point. Thank you. Let me clarify it.